PixivModelFanbox.py

```diff
@@ -159,14 +162,11 @@ def __str__(self):
     def parsePost(self, jsPost):
         self.imageTitle = jsPost["title"]
 
-        self.coverImageUrl = jsPost["coverImageUrl"]
+        coverUrl = jsPost["coverImageUrl"]
         # Issue #930
-        if self.coverImageUrl is not None:
-            original_url = jsPost["coverImageUrl"]
-            self.fullCoverSizeUrl = _re_fanbox_cover.sub("fanbox", original_url)
-            self.coverImageUrl = self.fullCoverSizeUrl
-            # self.try_add(self.fullCoverSizeUrl, self.images)
-            # self.try_add(self.fullCoverSizeUrl, self.embeddedFiles)
+        if not self.coverImageUrl and coverUrl:
```

Comment:
If the self.coverImageUrl already set, then it won't set the self.fullCoverSizeUrl anymore.

Reply:
i removed the `fullCoverSizeUrl` field, and set the `coverImageUrl` to the full sized image url inside the if. =)

Reply:
ok, looks like `coverImageUrl` only set here, so there is no risk that it is already have some values and the big cover is not resolved